tests/api/rbac.test.js

```diff
@@ -3,26 +3,18 @@
 jest.retryTimes(global.retry)
 
 const squad = require('../../config').get('squadName')
-const { getSearchApiRoute } = require('../common-lib/clusterAccess')
+const { getUserContext, getSearchApiRoute } = require('../common-lib/clusterAccess')
 const { ValidateSearchData, validationTimeout } = require('../common-lib/validateSearchData')
 const { execSync } = require('child_process')
 const { execCliCmdString } = require('../common-lib/cliClient')
+const { sleep } = require('../common-lib/sleep')
 
 const ns = 'search-rbac'
 const [usr0, usr1, usr2, usr3] = ['search-user0', 'search-user1', 'search-user2', 'search-user3']
 
-function buildUserObject(userName) {
-  return {
-    fullName: `system:serviceaccount:${ns}:${userName}`,
-    name: userName,
-    namespace: ns,
-    token: execSync(`oc serviceaccounts get-token ${userName} -n ${ns}`),
-  }
-}
-
 describe(`[P2][Sev2][${squad}] Search API: Verify RBAC`, () => {
   beforeAll(async () => {
-    // Usng ServiceAccounts for these tests because somfiguration is simpler.
+    // Usng ServiceAccounts for rbac tests because configuration is simpler.
```

Comment:
```suggestion
    // Using ServiceAccounts for rbac tests because configuration is simpler.
```

Reply:
This can be updated in another PR.

Reply:
will fix in my next PR branch to avoid having to wait for prow.